README.rst

```diff
@@ -10,6 +10,7 @@ Citation
 --------
 
 Cite NeuroR with the following DOI:
+
    .. image:: https://zenodo.org/badge/244944511.svg
    :target: https://zenodo.org/badge/latestdoi/244944511
```

Comment:
This should be indented one more level than the line above. See rendered output here: https://github.com/BlueBrain/NeuroR/tree/jdcourcol-patch-1